enabler/src/de/schildbach/pte/AbstractNavitiaProvider.java

```diff
@@ -208,6 +207,20 @@ protected String getLocationName(String name) {
         return name;
     }
 
+    /**
+     * Navitia always formats the address label in the French way.
+     * This method can be used to display addresses in a localized way.
+     * It defaults to putting the house number behind the street name.
+     *
+     * @param name The name of the address. Usually a street name.
+     * @param houseNumber The house number of the address. "0" if none available.
```

Comment:
Personally I'd prefer to handle the '0' special case outside of this method and use `null` instead.

Reply:
How about just calling this method if there is a non `null` or `"0"` housenumber?

Reply:
Would be fine to me too.

Reply:
All I want to avoid is bleed a broken API (use '0' if you mean 'none') into our own code.